vividus-plugin-mobile-app/src/main/resources/spring.xml

```diff
@@ -92,6 +94,8 @@
 
     <bean class="org.vividus.ui.mobile.action.MobileAppElementActions" />
 
+    <bean class="org.vividus.mobileapp.action.NetworkActions" />
```

Comment:
not sure if there will be any injection of this dependency other than network steps, so I think it makes sense to make it an inner bean